ssv/spectest/tests/valcheck/valcheckduty/far_future_duty_slot.go

```diff
@@ -29,6 +29,13 @@ func FarFutureDutySlot() tests.SpecTest {
 	return &valcheck.MultiSpecTest{
 		Name: "far future duty slot",
 		Tests: []*valcheck.SpecTest{
+			{
+				Name:       "committee",
+				Network:    types.BeaconTestNetwork,
+				RunnerRole: types.RoleCommittee,
+				Input:      testingutils.TestBeaconVoteByts,
+				// No error since input doesn't contain slot
```

Comment:
aite
I think I omitted this on purpose but maybe what you're doing is better?

Reply:
I think it's worth adding tests for the committee role here, so we can compare how each runner behaves